Import/READ_ME.txt

```diff
@@ -73,16 +73,11 @@ Installation Instructions
 		Select this manifest File "…\3-Collaboration Folders\2-imports (admin).mf"
 		"Collaboration Folders" <-- place check mark in this option and start the import.
 
-	(Optional) 4-Collaboration Folders on TOC:
+	4-Collaboration Folders on TOC:
 		Select this manifest File "…\4-Collaboration Folders on TOC\imports (admin).mf"
 		"Collaboration Folders" <-- place check mark in this option and start the import.
 
-	(Optional) 5-Configuration Data:
```

Comment:
The order of the import packages should remain the same, even if some imports are optional.

Reply:
Please disregard this comment. It seems there are issues with the optional imports, regardless of the order that they're imported. There is already another issue filed (#7) to handle these import issues and we are contacting the original project author to resolve those issues. Your changes to the package order will be fine in the meantime.